app/views/how-tos/passing-data-page.html

```diff
@@ -10,91 +10,100 @@
 
-      <pre class="app-pre"><code>docs/views/examples/passing-data</code></pre>
 
       <h2>How to use</h2>
 
-      <p>The kit stores data from inputs using the name attribute of the input.</p>
+      <p>When a user answers questions, their answer is saved within the 'data' object using the <code>name</code> attribute of the input.</p>
 
-      <p>For example, if you have this input:</p>
+      <p>For example, if you use this input to collect a user’s name:</p>
 
-      <pre class="app-pre"><code class="language-markup">&lt;input name="first-name"&gt;</code></pre>
+      <pre class="app-pre"><code class="language-markup">{% raw %}{{ input({
+  label: {
+    text: "Full name"
+  },
+  id: "full-name",
+  name: "fullName"
+}) }}
+{% endraw %}</code></pre>
 
       <p>You can show what the user entered later on like this:</p>
 
-      <pre class="app-pre"><code class="language-markup">&lt;p&gt;&#123;&#123; data&#91;'first-name'&#93; &#125;&#125;&lt;/p&gt;</code></pre>
-
-      <h3>Showing answers in inputs</h3>
-      <p>If a user goes back to a page where they entered data, they would expect to see the answer they gave.</p>
+      <pre class="app-pre"><code class="language-markup">{% raw %}<p>{{ data.fullName }}</p>{% endraw %}</code></pre>
 
-      <p>For a text input:</p>
+      <p>Answers from checkboxes will appear with commas, like 'a,b,c'. To show them as a list, use a <code>for</code> loop:</p>
 
-      <pre class="app-pre"><code class="language-markup">&lt;input name="first-name" value="&#123;&#123; data&#91;'first-name'&#93; &#125;&#125;"&gt;</code></pre>
+      <pre class="app-pre"><code class="language-markup">{{"<ul>
+  {% for condition in data.conditions %}
+    <li>{{ condition }}</li>
+  {% endfor %}
+</ul>" | escape }}</code></pre>
 
-      <p>For a radio or checkbox input you need to use the "checked" function:</p>
-
-      <pre class="app-pre"><code class="language-markup">&lt;input name="know-nhs-number" type="radio" value="Yes" &#123;&#123; checked("know-nhs-number", "Yes") &#125;&#125;&gt;</code></pre>
+      <h3>Showing answers in inputs</h3>
 
-      <h3>Clearing data</h3>
-      <p>To clear the data (for example, at the end of a user research session) click the "Clear data" link in the footer.</p>
+      <p>If a user goes back to a page where they entered data, they would expect to see the answer they gave.</p>
 
+      <p>Most inputs use the <code>value</code> option:</p>
 
-      <h3>Using the data in Nunjucks macros</h3>
+      <pre class="app-pre"><code class="language-markup">{% raw %}{{ input({
+  label: {
+    text: "Full name"
+  },
+  id: "full-name",
+  name: "fullName",
+  value: data.fullName
+}) }}
+{% endraw %}</code></pre>
 
-      <p>Example using the "checked" function in a checkbox component macro:</p>
+      <p>For a radios and checkboxes you need to use the "checked" filter:</p>
```

Comment:
Once this PR has been merged and released: https://github.com/nhsuk/nhsuk-frontend/pull/1105 then this section could be simplified by setting `values: data.conditions` instead of having to use the `checked` filter.